R/get.venn.partitions.R

```diff
@@ -0,0 +1,159 @@
+get.venn.partitions <- function(x, force.unique = TRUE, keep.elements = TRUE, hierarchical = FALSE) {
```

Comment:
Seems like this script and `make.truth.table.R` are splitted from the original `partitionText.R`?